_docs/platforms/kubernetes/install/1.10.md

```diff
@@ -27,20 +21,29 @@ or using the Helm chart:
 git clone https://github.com/storageos/helm-chart.git storageos
 cd storageos
 # Set cluster.join to hostnames or ip addresses of at least one node
-helm install . --name my-release --set cluster.join=node01,node02,node03 --set csi.enable=true
+helm install . --name my-release --set cluster.join=node01,node02,node03
+
+# Follow the instructions printed by helm install to update the link between Kubernetes and StorageOS. They look like:
+$ ClusterIP=$(kubectl get svc/storageos --namespace storageos -o custom-columns=IP:spec.clusterIP --no-headers=true)
+$ ApiAddress=$(echo -n "tcp://$ClusterIP:5705" | base64)
+$ kubectl patch secret/storageos-api --namespace storageos --patch "{\"data\":{\"apiAddress\": \"$ApiAddress\"}}"
 ```
 
-## Non-CSI installs
+## Installation with CSI (Container Storage Interface)
 
 CSI allows you to set StorageOS features (`storageos.com/*` labels) in
 the StorageClass, but not PVCs. If you need to set labels in PVCs (but not
 StorageClasses) or your environment does not support CSI, you may install
 StorageOS without CSI:
 
+CSI will become the standard for storage providers at Kubernetes 1.12 (CSI)](https://github.com/container-storage-interface/spec) 
+Using CSI ensures forward compatibility with future releases of Kubernetes, as
+vendor-specific drivers will be deprecated from Kubernetes 1.12.
```

Comment:
I'd remove this line.  The drivers are part of the API so they won't be deprecated until 2.0.  A shim will probably be put in place before then, but I don't think that made it into 1.12.